pyvista/plotting/text.py

```diff
@@ -253,44 +262,98 @@ class Label(Text):
 
     Examples
     --------
-    Create a label for a point of interest.
+    Create a label for a point of interest. Here we add a label to the tip of a cone.
 
     >>> import pyvista as pv
-    >>> mesh = pv.Cone()
-    >>> tip_point = mesh.points[0]
-    >>> label = pv.Label('tip', position=tip_point)
+    >>> cone_dataset = pv.Cone()
+    >>> tip = (0.5, 0, 0)
+    >>> label = pv.Label('tip', position=tip)
 
     Plot the mesh and label.
 
     >>> pl = pv.Plotter()
-    >>> _ = pl.add_mesh(mesh)
+    >>> cone_actor = pl.add_mesh(cone_dataset)
+    >>> _ = pl.add_actor(label)
+    >>> pl.show()
+
+    The previous example set the label's position as the cone's tip explicitly.
+    However, this means that the two actors now have different positions.
+
+    >>> cone_actor.position
+    (0.0, 0.0, 0.0)
+    >>> label.position
+    (0.5, 0.0, 0.0)
+
+    And if we change the orientation, the label is no longer positioned at the tip.
+
+    >>> cone_actor.orientation = 10, 20, 30
+    >>> label.orientation = 10, 20, 30
+    >>>
+    >>> pl = pv.Plotter()
+    >>> _ = pl.add_actor(cone_actor)
     >>> _ = pl.add_actor(label)
     >>> pl.show()
+
+    If we want the position of the label to have the same *relative* position to the
+    cone, we can set its :attr:`relative_position` instead.
+
+    First, set the label's position to match the cone's position.
+
+    >>> label.position = cone_actor.position
+    >>> label.position
+    (0.0, 0.0, 0.0)
+
+    Now set its :attr:`relative_position`.
+
+    >>> label.relative_position = tip
+    >>> label.relative_position
+    (0.5, 0.0, 0.0)
+
+    Plot the results.
+
+    >>> pl = pv.Plotter()
+    >>> _ = pl.add_actor(cone_actor)
+    >>> _ = pl.add_actor(label)
+    >>> pl.show()
+
```

Comment:
Usefulness of this function will become apparent after you move the cone in my mind. Can you add an explanation and code of that as well?

Reply:
The example is updated in https://github.com/pyvista/pyvista/pull/6353/commits/b8eaaeae34c9874f9c37ec5615a36d14335ebe5c, hopefully the utility of `relative_position` is clearer now.